evm_trace/gas.py

```diff
@@ -1,54 +1,81 @@
-from typing import Dict, List
+from typing import Any, Dict, List
+
+from hexbytes import HexBytes
+from pydantic import BaseModel, ValidationError, validator
 
 from evm_trace.base import CallTreeNode
+from evm_trace.utils import _convert_hexbytes
+
+Report = Dict[str, Dict[str, List[int]]]
+
+
+class Address(BaseModel):
+    address: Any
```

Comment:
Should the type here be `HexBytes`?
Same with `method_id` below?

Reply:
You comment reminded me that we had the ethpm-types HexBytes proof of concept. I've added it here to test it further. [a72b3f1](https://github.com/ApeWorX/evm-trace/pull/14/commits/a72b3f12de70f0c1ec75836fa6b8a3f422f29d56)